scripts/build_release.py

```diff
@@ -37,7 +37,8 @@
     "azure:westeurope" : "dxCompiler_Amsterdam",
     "aws:eu-central-1" : "dxCompiler_Berlin",
     "aws:eu-west-2": "dxCompiler_London",
-    "aws:eu-west-2-g": "dxCompiler_Europe_London"
+    "aws:eu-west-2-g": "dxCompiler_Europe_London",
+    "azure:uksouth-ofh": "dxCompiler_OFH_TRE"
```

Comment:
How about we name it dxCompiler_OFH_TRE_London (in case there will be a new region in the future for OFH-TRE)?

Reply:
Sure

Reply:
fixed in [dd4730a](https://github.com/dnanexus/dxCompiler/pull/407/commits/dd4730a369b0c731832d98415aa7a5be7e02e0b2)